examples/tests/sdk_tests/bucket/signed_url.test.w

```diff
@@ -69,7 +69,12 @@ test "signedUrl PUT" {
 }
 
 test "signedUrl duration option is respected" {
-  let ACCESS_DENIED_ERROR = "<Error><Code>AccessDenied</Code><Message>Request has expired</Message>";
+  let isExpiredTokenError = (output: str) => {
+    let awsError = output.contains("<Code>AccessDenied</Code><Message>Request has expired</Message>");
+    let gcpError = output.contains("<Code>ExpiredToken</Code><Message>Invalid argument.</Message>");
+    return awsError || gcpError;
+  };
```

Comment:
we can maybe condition it on the `util.env("WING_TARGET")`? (within an object or a switch-case maybe?)